enterprise/app/api_keys/api_keys.tsx

```diff
@@ -596,25 +597,53 @@ class ApiKeyField extends React.Component<ApiKeyFieldProps, ApiKeyFieldState> {
   state = ApiKeyFieldDefaultState;
 
   private copyTimeout: number | undefined;
+  private value: string | undefined;
+
+  componentDidMount() {
+    if (this.props.apiKey.value) {
+      this.value = this.props.apiKey.value
+    }
+  }
+
+  private async retrieveValue(fn: (val: string) => void) {
```

Comment:
nit: since you're using `async/await` here, this func will return `Promise<string>`, so you could write

```
return this.value
```

instead of

```
fn(this.value);
return;
```

then invoke the func like

```
const value = this.retrieveValue().then((val: string) => { ...});
```

instead of passing in a callback arg.

Reply:
Done